src/ui/buttons/buttonDesigns/alternateSlideLogoDesign.jsx

```diff
@@ -73,21 +77,29 @@ export const getAlternateSlideLogoProps = function(document : Object, configurat
     const paypalLabelContainerElement = designContainer.querySelector(`.${ PAYPAL_BUTTON_LABEL }`) || null;
     // get starting position for element so it doesn't jump when animation begins
     const paypalLogoElement = (paypalLabelContainerElement && paypalLabelContainerElement.querySelector(`.${ PAYPAL_LOGO }`)) || null;
-    const paypalLogoStartingLeftPosition = paypalLogoElement
-        ? `${ (paypalLogoElement.offsetLeft / paypalLabelContainerElement.offsetWidth) * 100 }`
-        : '44.5';
-
+    
+    if (paypalLogoElement && paypalLabelContainerElement) {
+        paypalLogoStartingLeftPosition = `${ (paypalLogoElement.offsetLeft / paypalLabelContainerElement.offsetWidth) * 100 }`;
+    } else {
+        return null;
+    }
+    // increase font-size for medium button sizes
+    if  (animationContainerWidth >= smalMax)  {
```

Comment:
Are the fontSize changes required by the design person? Can we handle this with a ternary instead?

Reply:
yeah I am okay with that